package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "mercurius",
-  "version": "13.3.0",
+  "version": "13.3.1",
```

```suggestion
  "version": "13.3.0",
```

We'll bump the version accordingly on release